tests/BenchmarkDotNet.Tests/BenchmarkDotNet.Tests.csproj

```diff
@@ -42,4 +42,10 @@
   <ItemGroup>
     <EmbeddedResource Include="Portability\Cpu\TestFiles\*.txt" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
It seems that these lines were committed by mistake: I don't see any reasons to exclude the declared files from the project structure.